tests/msgpack.tests/ObjectCreatorPerformanceTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+using Shouldly;
+
+using Xunit;
+
+namespace TarantoolDnx.MsgPack.Tests
+{
+    public class ObjectCreatorPerformanceTest
+    {
+        private const int TestsCount = 10000000;
+
+        [Fact]
+        public void TestActivator()
```

Comment:
This perfomance test is bad. We just slow down build and don't get any useful info